### ISSUE TITLE
When will the project be updated?

### ISSUE THREAD

**Issue by 1110680** [CognitionFoundry/gohfc#4]
can’t wait

**Comment by gatakka** [comment]
When Hyperledger Fabric 1.1 is officially released I will push the update of the SDK.
This is because 1.1 is a very big update that break some backward compatibility.
After version 1.1 this project will be move to agile type of development.

**Comment by bogdanprodanj** [comment]
Thanks a lot for the update!!!

**Comment by gatakka** [comment]
Yes, I know that. Try to setup Java SDK, the setup/configuration size is comparable to the size of this SDK.
